packages/runtime/src/helpers/edge.ts

```diff
@@ -4,7 +4,7 @@ import { resolve, join } from 'path'
 import type { NetlifyConfig, NetlifyPluginConstants } from '@netlify/build'
 import { greenBright } from 'chalk'
 import destr from 'destr'
-import { copy, copyFile, emptyDir, ensureDir, readJSON, readJson, writeJSON, writeJson } from 'fs-extra'
+import { copy, copyFile, emptyDir, ensureDir, readJSON, writeJSON, writeJson } from 'fs-extra'
```

Comment:
I just notice there’s the same function name twice in fs-extra but with different casings. Odd.

Reply:
Yeah, I think that it's a legacy thing. Confusing, as I'm not sure which is "preferred"